activetable/templates/html/activetable.html

```diff
@@ -0,0 +1,47 @@
+<div class="activetable_block">
+  {% if help_text %}
+  <div class="activetable-help" style="width: {{ total_width }}px;">
+    <button id="activetable-help-button">+help</button>
```

Comment:
@cptvitamin I added the `aria-…` properties you mentioned.

And you are right, we currently don't support multiple tables per page.  Some of the ids could easily be turned into classes.  Most lookups are performed relative to the top-level XBlock div anyway, so we wouldn't get conflicts with other tables on the same page.  However, the `p` element with the help text needs to be referenced by id in the `aria-controls` label, so turning it into a class is not an option.  I guess I will have to add some hashes to the few ids we need, and pass them around everywhere.